nerfstudio/engine/schedulers.py

```diff
@@ -114,3 +114,38 @@ def __init__(
             max_steps,
         )
         super().__init__(optimizer, lr_init, lr_final, max_steps, delay_epochs, after_scheduler=after_scheduler)
+
+
+@dataclass
```

Comment:
Setting up schedulers are a mess. Im going update the scheduler code to match how configs work elsewhere. Then this can be refactored.

Reply:
https://github.com/nerfstudio-project/nerfstudio/pull/1466